src/deepsparse/benchmark_model/ort_engine.py

```diff
@@ -0,0 +1,309 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+from typing import Dict, Iterable, List, Optional, Tuple, Union
+
+import numpy
+
+from deepsparse.benchmark import BenchmarkResults
+from deepsparse.utils import (
+    get_input_names,
+    get_output_names,
+    model_to_path,
+    override_onnx_batch_size,
+    override_onnx_input_shapes,
+)
+
+
+try:
+    import onnxruntime
+
+    ort_import_error = None
+except Exception as ort_import_err:
+    onnxruntime = None
+    ort_import_error = ort_import_err
+
+try:
+    from sparsezoo import Zoo
```

Comment:
why do we have a try around the zoo imports? These should be always available, correct? I'd be worried this hiding issues that shouldn't be happening

Reply:
this is copied from engine.py, do you know why it was originally setup that way?

Reply:
Following up here since we discussed it in slack. I believe it was in there due to nyann internal repo not installing deepsparse properly. Would prefer if we fixed the nyann flows, but if not in scope then good with landing this as is or preferably removing the try except here and forcing nyann to migrate to better flows when using this script